### ISSUE TITLE
Can't run the web version demo

### ISSUE THREAD

**Issue by azerphoenix** [mapsforge/vtm#774]
Hello.
I try to run a web demo via ` ./gradlew :vtm-web-app:farmRun`
Getting an exception
`SDK location not found. Define location with an ANDROID_SDK_ROOT environment variable or by setting the sdk.dir path in your project's local properties file at '/home/phoenix/IdeaProjects/vtm/local.properties'.`
Then created local.properties and added sdk.dir to /home/phoenix/Android/Sdk

Tried once more to start via `./gradlew :vtm-web-app:farmRun`
Getting exceptions:
```
java.lang.StringIndexOutOfBoundsException
java.lang.IllegalStateException
```
And the compiling freezes at 97 %.
Full log:
[https://pastebin.com/WeLvaU24](https://pastebin.com/WeLvaU24)
How I can fix this?

OS - Linux Mint 64 bit
JDK - AdoptOpenJDK 8 & 11 (installed via sdkman)

Thanks in advance

**Comment by devemux86** [comment]
In the past it worked fine, so you can try an older Gradle version.

_In any case OpenScienceMap tile server does not work anymore._

**Comment by azerphoenix** [comment]
Thank you so much for the quick reply!
Just tried to run using the following versions of gradle 4.0, 5.0, 6.3. And get the same exceptions.
May I ask you - is there a demo of integration of VTM with spring boot? Thanks in advance.
Created question in Google Groups - https://groups.google.com/forum/#!topic/mapsforge-dev/I93diMYKR9k

**Comment by devemux86** [comment]
Fixed via #780.
